docs/docs/glossary.mdx

```diff
@@ -17,6 +17,11 @@ The server that runs custom action code, separate from Rasa. Rasa maintains the
 
 Adding labels to messages and conversations so that they can be used to train a model.
 
+## [Anonymization](./pii-management.mdx)
+
+The process of replacing personally identifiable information (PII) with masked, artificial or constant text values.
```

Comment:
I saw a review note on this in my email notifications, but it doesn't seem to be in the GH UI anymore.
In any case, could you please pseudo-anonymization? Is that the same as replacing real PII with fake generated data? In which case I mention here as `artificial`, so I consider this to be a subset of anonymization. Let me know how I should rephrase to make it clearer.